src/ui.rs

```diff
@@ -0,0 +1,109 @@
+use bevy::{prelude::*, utils::HashSet};
+use bevy_egui::{
+    egui::{self},
+    EguiContexts, EguiPlugin,
+};
+
+use crate::animation_loader::{AnimationSelectEvent, BvhFile};
+
+pub struct UiPlugin;
+
+impl Plugin for UiPlugin {
+    fn build(&self, app: &mut App) {
+        app.add_systems(Update, ui)
+            .add_plugins(EguiPlugin)
+            .insert_resource(ShowDrawArrow { show: true })
+            .insert_resource(SelectedFiles::default())
+            .add_event::<DrawArrowEvent>();
+    }
+}
+
+#[derive(Resource)]
+pub struct ShowDrawArrow {
+    pub show: bool,
+}
+
+#[derive(Resource, Default, Debug)]
+pub struct SelectedFiles {
+    pub files: HashSet<String>,
+}
+
+#[derive(Event, Debug)]
+pub struct DrawArrowEvent(pub bool);
+
+pub fn animation_files_menu(
+    ui: &mut egui::Ui,
+    file_name: &ResMut<BvhFile>,
+    mut event_writer: EventWriter<AnimationSelectEvent>,
+) {
+    ui.horizontal(|ui| {
+        ui.label("Choose Animation File:");
+        egui::ComboBox::from_label("").show_ui(ui, |ui| {
+            for file in file_name.0.iter() {
+                if ui.selectable_label(false, file).clicked() {
+                    event_writer.send(AnimationSelectEvent(file.to_string()));
+                }
+            }
+        });
+    });
+}
+
+pub fn build_button(ui: &mut egui::Ui) {
+    if ui.button("Build").clicked() {
+        println!("Build Buttonnnnnnnnnnnnnnnnnnnn");
+    }
+}
+
+pub fn arrow_checkbox(
+    ui: &mut egui::Ui,
+    mut show_draw_arrow: ResMut<ShowDrawArrow>,
+    mut event_writer: EventWriter<DrawArrowEvent>,
+) {
+    if ui
+        .checkbox(&mut show_draw_arrow.show, "Show Arrows")
+        .changed()
+    {
+        event_writer.send(DrawArrowEvent(show_draw_arrow.show));
+    }
+}
```

Comment:
`ShowDrawArrow` resource is sufficient.
```suggestion
pub fn arrow_checkbox(
    ui: &mut egui::Ui,
    mut show_draw_arrow: ResMut<ShowDrawArrow>,
) {
    ui.checkbox(&mut show_draw_arrow.show, "Show Arrows");
}
```